<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAttribute</class>
 <widget class="QDialog" name="NewAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouveau attribut</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="attributeName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="attributeType">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> `&lt;typename&gt;`</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="editType">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="attributeRange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Publique</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Protégé</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Privé</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="attributeStatic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Statique</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="attributeTransient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Non-sérialisé</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Exclusif Java</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>java.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="validate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminer</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
